Dockerfile

```diff
@@ -1,10 +1,20 @@
-FROM alpine:latest
+FROM ubuntu:16.04 as build
+
+ENV GOPATH=/go
+WORKDIR /go/src/app
+RUN apt-get update && apt-get install -y openssl libssl-dev curl git build-essential pkg-config golang
+
+RUN go get golang.org/dl/go1.15
+RUN /go/bin/go1.15 download
+RUN ln -sf /go/bin/go1.15 /usr/bin/go
 
-RUN apk update && apk add --no-cache openssl
+COPY . .
 
-RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
+RUN make buildstatic
+
+FROM alpine:latest
```

Comment:
Current when done, not touched it since. 

Sorry for the delay, I have not worked more on it since my original commit and the original spike was paused for other things back then.